neurons/validator.py

```diff
@@ -54,48 +54,32 @@ async def run_step(self, k: int, timeout: float) -> ValidatorLoggingEvent | Erro
         if len(scoring_queue) > SCORING_QUEUE_LENGTH_THRESHOLD:
             logger.debug("Scoring queue is full. Skipping task generation.")
             return None
+        if len(mutable_globals.task_queue) == 0:
+            logger.warning("No tasks in queue, skipping sending...")
+            return
         try:
-            # Getting task & Dataset
+            # get task from the task queue
+            mutable_globals.task_queue: list[BaseTextTask]
```

Comment:
Why do we need to redeclare it here?
Not related to this PR, but we need to get rid of mutable global vars, as it's considered to be a bad practice in general

Reply:
Do you have any suggestion for how else to handle the queues? I think they do need to be defined globally, otherwise you run into a ton of circular import issues. The reason why I'm redeclaring is to get correct type hinting